### DISCUSSION TITLE
Problems with custom templates

### DISCUSSION THREAD

**Discussion by Ma1Dong** [projectdiscovery/nuclei-templates#4271] [Q&A] [answered]
I want the fields in the response body to be displayed in the DOS window，What should I do，thinks

For example，I want this version information，In the result of my DOS window
![图片](https://user-images.githubusercontent.com/52943794/166094363-cc0b51cd-3a19-45c2-b848-a2e466255e89.png)
![图片](https://user-images.githubusercontent.com/52943794/166094432-ad814bbd-ae04-47cc-a75f-62a2a83641b6.png)

**Comment by ehsandeep** [comment, marked as answer]
@Ma1Dong, you can use [extractors](https://github.com/projectdiscovery/nuclei-templates/search?q=extractors) on the top of matchers to print additional data.

**Reply by Ma1Dong** [reply]
thinks
